test/support/slack_client.ex

```diff
@@ -73,9 +73,39 @@ defmodule Cog.Test.Support.SlackClient do
       nil ->
         raise RuntimeError, message: "$SLACK_USER_API_TOKEN not set!"
       token ->
-        {:ok, client} = start_link(token)
-        :ok = initialize(client)
-        {:ok, client}
+        api_wait()
+        api_wait()
+        try do
+          {:ok, client} = start_link(token)
+          :ok = initialize(client)
+          {:ok, client}
+        rescue
+          e in RuntimeError ->
```

Comment:
Eh, I think we'll be ok since we're logging it at least. Just wanted to avoid hiding any actual bugs behind this retry code. I think we'll be ok though.